docs/code-search/code-navigation/how-to/adding_lsif_to_many_repos.mdx

```diff
@@ -1,4 +1,4 @@
-# Tutorial: Adding precise indexing to many repositories
```

Comment:
This page is quite out-of-date; let's delete it instead of making tweaks.

Reply:
Ok removing it altogether.